sycl/include/sycl/detail/pi.h

```diff
@@ -290,6 +293,9 @@ typedef enum {
   // Return 0 if device doesn't have any memory modules. Return the minimum of
   // the bus width values if there are several memory modules on the device.
   PI_EXT_INTEL_DEVICE_INFO_MEMORY_BUS_WIDTH = 0x10031,
+  // Return 1 if the device doesn't have a notion of a "queue index". Otherwise,
```

Comment:
why "1" and not "0"?

Reply:
Per extension specification. The idea was that returning "1" would mean that the only possible index is "0" and it can be ignored on backends/devices that don't support partitioning, making the extension itself supported everywhere.

Reply:
OK